src/App.tsx

```diff
@@ -15,36 +17,100 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+let goods = [...goodsFromServer];
+let isReverseCLicked = false;
```

Comment:
[CODE KNOWLEDGE] It's not a good practice to use global variables to manage state in a React component. Consider moving `goods` and `isReverseCLicked` inside `App` component and manage their state using `useState`.